mon-pix/app/components/routes/login-form.js

```diff
@@ -64,9 +64,9 @@ export default class LoginForm extends Component {
     const scope = 'mon-pix';
     try {
       await this.session.authenticate('authenticator:oauth2', { login, password, scope });
-    } catch (err) {
-      const title = ('errors' in err) ? err.errors.get('firstObject').title : null;
-      if (title === 'PasswordShouldChange') {
+    } catch (response) {
+      const isPasswordExpiredError = get(response, 'responseJSON.errors[0].title') === 'PasswordShouldChange';
+      if (isPasswordExpiredError) {
```

Comment:
Un peu de mal avec la notion d'expiration de password, je propose :

```suggestion
      const shouldChangePassword = get(response, 'responseJSON.errors[0].title') === 'PasswordShouldChange';
      if (shouldChangePassword) {
```

Est ça colle pas mal avec le `===`

Reply:
Ah je viens de comprendre d'où venait la confusion : le nom de la route c'est `update-expired-password`

Reply:
Je maintiens que je ne suis pas à l'aise avec la notion, mais c'est ok si on ne prend pas ma suggestion pour cette PR

Reply:
J'ai fait la modification